packages/dapp/src/components/MultisigConfirmationModal/MultisigConfirmationModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { Modal, Typography } from '@mui/material';
+import React from 'react';
```

Comment:
Importing react is not necessary anymore, please remove it everywhere.

Reply:
okay